packages/experimental-loader/src/create-runtime-target-code.ts

```diff
@@ -2,7 +2,7 @@ import { StylableExports } from '@stylable/core';
 
 export function createRuntimeTargetCode(namespace: string, mapping: StylableExports) {
     return `
-  var rt = require("@stylable/runtime/cjs/css-runtime-stylesheet.js");
+  var rt = require("@stylable/runtime");
```

Comment:
use "@stylable/runtime/dist/css-runtime-stylesheet.js" we don't want the renderer to be included

Reply:
Done.